extensions/aws/processors/PutS3Object.h

```diff
@@ -44,7 +44,10 @@ class PutS3Object : public S3Processor {
   static const std::set<std::string> STORAGE_CLASSES;
   static const std::set<std::string> SERVER_SIDE_ENCRYPTIONS;
 
-  EXTENSIONAPI static constexpr const char* Description = "This Processor puts FlowFiles to an Amazon S3 Bucket.";
+  EXTENSIONAPI static constexpr const char* Description = "Puts FlowFiles to an Amazon S3 Bucket. The upload uses either the PutS3Object method. "
```

Comment:
```suggestion
  EXTENSIONAPI static constexpr const char* Description = "Puts FlowFiles to an Amazon S3 Bucket. The upload uses the PutS3Object method. "
```

Reply:
fixed in 5331a2377aa7ca6b3b259ebabe6e88314480578d